Miscellaneous/Sign in with apple.md

```diff
@@ -0,0 +1,210 @@
+At WWDC 2019., Apple revealed **[Sign in with Apple - Apple Developer][1]**, a privacy and security focused login service. It provides users with a way to sign up or log in into application by leveraging the Apple ID they have set up on their devices.
+
+Initially it provided a nice alternative to Google/Facebook login, but as time passed, it became mandatory for all applications which are using any other type of third party login services, which is why it's necessary for you to know how to implement it. 
+
+Don’t fret, it’s actually quite simple, as you’ll be able to see after reading and following the steps below.
+
+## 1. Add the required capability
+
+The first thing you need to do is to add the new **Sign in with Apple capability**. To do so, you need to:  
+* click on your app target,
+* go to the *Signing and capabilities* tab,
+* click on the *+ Capability* button, 
+* double click on *Sign in with Apple*
+
+![Sign in with Apple capability][image-1]
+
+This capability is **available only for non-enterprise account types**, which means that in 99% of cases you can only add this capability to the production target/configuration (depending on your project setup) which is **signed using the client’s account, not Infinum D.O.O.**. `#if APPSTORE` or whatever you have configured in your project for differentiating environments will be your friend when following the next steps.
```

Comment:
I'm not sure if this is still relevant, but if it is, please add some kind of note here that in order to activate and use Sign in with Apple, the app should be signed with a client certificate (acc) and not with the Infinum d.o.o. The reason is that once the bundle ID of the app is registered on one account where Sign in with Apple capability is enabled, that app was "locked" to that account and we were not able to migrate it to the clients acc.